src/components/ReactTags.js

```diff
@@ -501,27 +503,29 @@ class ReactTags extends Component {
 
     const tagInput = !this.props.readOnly ? (
       <div className={classNames.tagInput}>
-        <input
-          {...inputProps}
-          ref={(input) => {
-            this.textInput = input;
-          }}
-          className={classNames.tagInputField}
-          type="text"
-          placeholder={placeholder}
-          aria-label={placeholder}
-          onFocus={this.handleFocus}
-          onBlur={this.handleBlur}
-          onChange={this.handleChange}
-          onKeyDown={this.handleKeyDown}
-          onPaste={this.handlePaste}
-          name={inputName}
-          id={inputId}
-          maxLength={maxLength}
-          value={inputValue}
-          data-automation="input"
-          data-testid="input"
-        />
+        {(!maxTags || (maxTags && tags.length < maxTags)) && (
```

Comment:
lets return null if max tags reached instead for better readability

Reply:
thanks for the suggestion :)